base/task.jl

```diff
@@ -543,6 +556,24 @@ end
     errormonitor(t::Task)
 
 Print an error log to `stderr` if task `t` fails.
+
+# Examples
+```julia-repl
+julia> a4() = error("task failed");
+
+julia> b = Task(a4);
+
+julia> errormonitor(b);
+
+julia> schedule(b);
+
+Unhandled Task ERROR: task failed
+Stacktrace:
+ [1] error(s::String)
+   @ Base ./error.jl:35
+ [2] a4()
+   @ Main ./REPL[1]:1
```

Comment:
commonly, they are all stacked on the same line as:
```suggestion
julia> a4() = error("task failed");

julia> Base._wait(schedule(errormonitor(Task(a4))))
Unhandled Task ERROR: task failed
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:35
 [2] a4()
   @ Main ./REPL[1]:1
```

Reply:
Or perhaps more canonically using the spawn macro:
```suggestion
julia> Base._wait(errormonitor(Threads.@spawn error("task failed")))
Unhandled Task ERROR: task failed
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:35
 [2] (::var"#5#6")()
   @ Main ./task.jl:501
```